### PR TITLE
Fix bugs with threads

### PR DESCRIPTION
**Pull request by Geometrically**

### CODE REVIEW

File: helpers/notifications.js [hunk 1]
```diff
@@ -134,10 +134,11 @@ export function groupNotifications(notifications, includeRead = false) {
   for (const notification of notifications.filter((notif) => includeRead || !notif.read)) {
     // Group notifications of the same thread or project id
     if (notification.body) {
+      console.log(notification.body)
```
Comment (triphora) [inline helpers/notifications.js:137]: console log

File: layouts/default.vue [hunk 2]
```diff
@@ -476,6 +476,10 @@ export default defineNuxtComponent({
     },
   },
   mounted() {
+    if (process.client && window) {
```
Comment (triphora) [inline layouts/default.vue:479]: are you sure it's safe to have client-side-only stuff like this?

File: pages/[type]/[id].vue [hunk 3]
```diff
@@ -373,22 +373,6 @@
...
-          <a href="https://docs.modrinth.com/docs/modpacks/playing_modpacks/" :target="$external()"
-            >our documentation</a
-          >
-          which provides instructions on using
-          <a href="https://atlauncher.com/about" :target="$external()" rel="noopener"> ATLauncher</a
-          >, <a href="https://multimc.org/" :target="$external()" rel="noopener">MultiMC</a>, and
-          <a href="https://prismlauncher.org" :target="$external()" rel="noopener">
-            Prism Launcher</a
-          >.
-        </div>
```
Comment (triphora) [inline pages/[type]/[id].vue:391]: this definitely isn't worth removing entirely imo - there's really no other way on the site to tell people *how* to install a modpack

**Review by triphora — CHANGES_REQUESTED** [review]

File: pages/[type]/[id].vue (hunk 3 above)
Comment (CodexAdrian) [reply pages/[type]/[id].vue:391]: Agreed, perhaps replace it with text directing the user to install the new Modrinth App. Modpacks are still in beta support, and people might not be aware theres an official first party solution now

**Review by CodexAdrian — CHANGES_REQUESTED** [review]
Emma on point here

File: pages/[type]/[id].vue (hunk 3 above)
Comment (Prospector) [reply pages/[type]/[id].vue:391]: yup, I'd have it say to [download the Modrinth App] to install modpacks & link to the documentation for how to download them with other launchers

File: layouts/default.vue (hunk 2 above)
Comment (Geometrically) [reply layouts/default.vue:479]: yes its only client side code, it affects scroll behaviour

**Review by triphora — APPROVED** [review]

**Review by CodexAdrian — APPROVED** [review]

**Pull request merged**